app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationsList.scss

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+.integration-empty {
+  margin-bottom: 0px !important;
```

Comment:
Using important it's a bad practice. Ehe reason you can't override a style is because in EmptyStatePage the styles are under the .container class, pull all the styles out of it